README.md

```diff
@@ -2,26 +2,126 @@
 
 This guestbook is a place for people who have taken [OpenSauced](https://opensauced.pizza)'s [Intro to Open Source course](https://github.com/open-sauced/intro) to take their first steps into contributing to open source.
 
-While there's an automated bot that will help you add yourself to the contributor's list via an issue, we recommend actually editing the `.all-contributorsrc` file to get more experience with pull requests and reviews.
```

Comment:
Instead of using "bot," let's use "CLI tool" throughout the readme.

Reply:
I've replaced the "bot" with "CLI".